src/main/java/mekanism/common/item/gear/ItemMekaSuitArmor.java

```diff
@@ -329,6 +331,26 @@ public boolean elytraFlightTick(ItemStack stack, LivingEntity entity, int flight
         return true;
     }
 
+    @Override
+    public boolean canUseJetpack(ItemStack stack) {
+        return slot == EquipmentSlot.CHEST && (isModuleEnabled(stack, MekanismModules.JETPACK_UNIT) && ChemicalUtil.hasChemical(stack, MekanismGases.HYDROGEN.get())
```

Comment:
Upon closer inspection I think this long line needs to be split into multiple if statements because if the jetpack is enabled but there is no chemical THEN it will check the OR which will return true because the jetpack is enabled and exclusive for overriding jump.

Reply:
I fixed it, but should we consider adding a canUse function to module? It could be used for a couple things which require energy or something else, including the canUseJetpack method. It would put the logic in one place in case it ever changes too.

Reply:
I think it might be worth at least considering, but I think that will end up waiting for 1.19